components/support/Category.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from 'next/image'
+import career from '../../public/assets/category/career.svg'
+import healthcare from '../../public/assets/category/healthcare.svg'
+import financial from '../../public/assets/category/financial.svg'
+import self from '../../public/assets/category/selfdevelopment.svg'
+import family from '../../public/assets/category/family.svg'
+import children from '../../public/assets/category/childeducation.svg'
+import legal from '../../public/assets/category/legal.svg'
+import Link from 'next/link'
+
+interface Props {
+  type: string
+  count: number
+  clickHandler?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const category_list = [
+  { type: 'career', image: career, text: 'Career' },
+  { type: 'healthcare', image: healthcare, text: 'Healthcare' },
+  { type: 'financial', image: financial, text: 'Financial Assistance' },
+  { type: 'self', image: self, text: 'Self Development' },
+  { type: 'family', image: family, text: 'Family Services' },
+  { type: 'children', image: children, text: 'Children Education' },
+  { type: 'legal', image: legal, text: 'Legal' }
+]
+
+export default function Category(props: Props) {
+  const type = props.type
+  const category_ref = category_list.filter(
+    (category) => category.type === type
+  )[0]
```

Comment:
I think we should probably check to see that the's an element first.

If there's no element we should log an error and skip it.